src/CSharpExtensions.Analyzers/SymbolHelper.cs

```diff
@@ -103,16 +110,29 @@ public class MemberSymbolInfo
     {
         public ISymbol Symbol { get; }
         public string ExpectedName { get; }
+        public bool IsEnumWithValue { get; }
+        public object EnumConstantValue { get; }
 
 
         public MemberSymbolInfo(ISymbol symbol, string namePrefix)
         {
             Symbol = symbol;
             ExpectedName = namePrefix + symbol.Name;
+
+            if (symbol is IFieldSymbol ff && ff.DeclaringSyntaxReferences[0].GetSyntax() is EnumMemberDeclarationSyntax e)
```

Comment:
Because the property `ConstantValue` is of type `object`, it can be anything, even `null`. It also contains the implicit default value of the enum.
`IsEnumWithValue` is therefor added to detect that the twin actually has values set.